lib/internal/util.js

```diff
@@ -431,6 +433,39 @@ function createDeferredPromise() {
   return { promise, resolve, reject };
 }
 
+let internalTimers;
+
+function getActiveResources() {
+  if (internalTimers == null) {
+    internalTimers = require('internal/timers');
+  }
+  const handles = process._getActiveHandles();
+  const reqs = process._getActiveRequests();
```

Comment:
Could you avoid using the global `process` here and instead load them from `internalBinding('process_methods')`?

Reply:
Thanks for noticing. :)
Replaced it.